cmake/projects/LAPACK/hunter-msvc.cmake

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2017, NeroBurner
+# All rights reserved.
+
+#hunter_cacheable(LAPACK) # install is always cacheable
+
+# determine architecture
+include(hunter_check_toolchain_definition)
+hunter_check_toolchain_definition(NAME "_WIN64"    DEFINED WIN64_defined)
```

Comment:
Please use `_WIN64_defined` or `_hunter_win64_defined` here.

Reply:
Add `hunter_report_broken_package` if `_WIN64` is not defined, example:
* https://github.com/ruslo/hunter/blob/d7c2b1da4423077c46fd1e89bd1e25120666ed78/cmake/projects/CLAPACK/hunter.cmake#L13